ecs-agent/manageddaemon/managed_daemon.go

```diff
@@ -80,11 +82,60 @@ func NewManagedDaemon(
 	return newManagedDaemon
 }
 
+var ImportAll = defaultImportAll
+
 // ImportAll function will parse/validate all managed daemon definitions
 // defined in /var/lib/ecs/deps/daemons and will return an array
 // of valid ManagedDeamon objects
-func ImportAll() ([]*ManagedDaemon, error) {
-	return []*ManagedDaemon{}, nil
+func defaultImportAll() ([]*ManagedDaemon, error) {
+	// TODO parse taskdef json files in parameterized dir ie /deps/daemons
+	// TODO validate that each daemon's layers are loaded or that daemon has a corresponding image tar
+	ebsCsiTarFile := filepath.Join(imageTarPath, "ebs-csi-driver", "ebs-csi-driver.tar")
+	if _, err := os.Stat(ebsCsiTarFile); err != nil {
+		return []*ManagedDaemon{}, nil
```

Comment:
Should we add a warning log here?

Reply:
This is an expected result --
If there are no managed daemons tars in the import dir, then the import all should return an empty array.
Ultimately `ImportAll` will check for a <daemon>.json daemon config and a corresponding <daemon>.tar image file and will import whichever have both.